### PR TITLE
Created a default TimerInteraction in both the DSL and in the Runtimes.

### PR DESCRIPTION
**Pull request by Tim-Linschoten**

### CODE REVIEW

File: core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/defaultinteractions/TimerInteraction.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import cats.effect.{IO, Timer}
+
+import scala.concurrent.duration.FiniteDuration
+
+class TimerInteraction(skipWait: Boolean)(implicit timer: Timer[IO]) {
+
+  class TimeWaited()
+
+  def apply(WaitTime: FiniteDuration): IO[TimeWaited] = {
```
Comment (jacum) [inline core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/defaultinteractions/TimerInteraction.scala:11]: I wonder if two implementations are necessary, can't we reduce code duplication?

File: core/baker-interface/src/main/scala/com/ing/baker/runtime/inmemory/InMemoryBaker.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ object InMemoryBaker {
     recipeInstanceManager <- InMemoryRecipeInstanceManager.build
     recipeManager <- InMemoryRecipeManager.build
     eventStream <- InMemoryEventStream.build
-    interactions <- InMemoryInteractionManager.build(implementations)
+    interactions <- InMemoryInteractionManager.build(implementations ++ defaultinteractions.getDefaultInteractions())
```
Comment (jacum) [inline core/baker-interface/src/main/scala/com/ing/baker/runtime/inmemory/InMemoryBaker.scala:21]: no () needed, getDefaultInteractions isn't effectful

**Review by jacum — COMMENTED** [review]
Should we add a spec covering that timer behaviour?

File: core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/defaultinteractions/TimerInteraction.scala (hunk 1 above)
Comment (Tim-Linschoten) [reply core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/defaultinteractions/TimerInteraction.scala:11]: Your are right, I was planning to create a specific one for the Akka runtime based on the schedular of Akka, but using the schedular of cats seems fine so I will remove the duplicates.

**Review by jacum — APPROVED** [review]

**Pull request merged**